python/pynessie/cli.py

```diff
@@ -186,6 +186,7 @@ def set_head(ctx: ContextObject, head: str, delete: bool) -> None:
 
 
 @cli.command("log")
+@click.option("-r", "--ref", help="branch to list from. If not supplied the default branch from config is used")
```

Comment:
I'd personally prefer to have it work like `git log branch-name HASH..HASH` (so two args instead of one arg plus an option) - not that new option.

Reply:
I was trying to make it consistent with other commands that use `--ref` but I don't have a personal preference. @rymurr thoughts on this?

Reply:
I don't have a strong opinion either. In doubt keep consistent so prob marginally lean towards the new ref option

Reply:
ok then let's move forward with the proposed approach with `--ref`